test/tests/functional/pbs_preemption.py

```diff
@@ -676,3 +676,33 @@ def test_preemption_priority_escalation_2(self):
         self.server.expect(JOB, {'job_state': 'S'}, id=jid_list[1])
         self.server.expect(JOB, {'job_state': 'S'}, id=jid_list[3])
         self.server.expect(JOB, {'job_state': 'S'}, id=jid_list[4])
+
+    @skipOnCpuSet
+    def test_preempt_requeue_static_resc(self):
+        """
+        Test that scheduler will preempt jobs for static resources
```

Comment:
Can you also mention RRTROS and preempt method R here ?

Reply:
Also, we don't really qualify static resources specially, we just call them resources and qualify dynamic resources instead right? I had to double check to make sure that "static resource" was just regular resources. So, I'd advice just calling it 'resource', but it's up to you :)